sdk/lib/convert/latin1.dart

```diff
@@ -63,12 +63,54 @@ class Latin1Codec extends Encoding {
 }
 
 /// This class converts strings of only ISO Latin-1 characters to bytes.
+///
+/// Example:
+/// ```dart
+/// const String sample = 'à á â ã ä å';
```

Comment:
Maybe we should remove the whitespace from sample so they are not converted to "32" and it's easier to spot 6 characters to convert to 6 different bytes